ext/standard/tests/strings/implode_error.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+--FILE--
+<?php
+/* only glue */
+try {
+    var_dump( implode("glue") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* int as pieces */
+try {
+    var_dump( implode("glue",1234) );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
```

Comment:
Please remove this, as this is a "standard" ZPP failure and we don't test those.

Reply:
Ok. I just copied from the _variation file. But I'll remove it. What does ZPP mean?

Reply:
ZPP stands for "Zend Parse Parameters" which is the name for the C API that parses parameters passed to functions and extract them as a C type :)